Library/Formula/shml.rb

```diff
@@ -0,0 +1,16 @@
+class Shml < Formula
+  desc "Style Framework for The Terminal"
+  homepage "http://shml.xyz"
+  url "https://github.com/MaxCDN/shml/archive/1.0.4.tar.gz"
+  sha256 "583fefbbf2adf45d989b052696be0f20f695903878558a4d2e9f74255b91ed51"
+
+  def install
+    install "shml.sh"
+    install "shml.sh" => "shml"
```

Comment:
You can't install the same file twice, because `.install` is essentially our helper for the Ruby version of `mv`.

The easiest way to deal with that sort of thing is to install the original somewhere, like `libexec.install "shml.sh"` and then symlink it. In shell terms:

```ruby
libexec.install "shml.sh"
```
would be more or less:
```bash
mv buildpath/shml.sh HOMEBREW_CELLAR/shml/version/libexec
```

And:
```ruby
bin.install_symlink libexec/"shml.sh" => "shml"
```
would be more or less:
```bash
ln -s HOMEBREW_CELLAR/shml/version/libexec/shml.sh HOMEBREW_CELLAR/shml/version/bin/shml
```